style.css

```diff
@@ -0,0 +1,519 @@
+  font-size: 24px;
+  align-self: center;
+  color: #1c1a19;
+}
+
+.line {
+  height: auto;
+  display: flex;
+  position: relative;
+  margin-left: 2.5%;
+  width: 100%;
+  margin-top: 2.5%;
+}
+
+.main-image {
+  display: flex;
+  margin-top: 15%;
+  width: 100%;
+  text-align: center;
+}
+
+.H2 {
+  margin-left: 0;
+  position: relative;
+  width: 90%;
+  margin-top: 5%;
+  font-size: 18px;
+  color: #1c1a19;
+  order: 0;
+}
+
+.p-recent > p {
+  position: relative;
+  width: 80%;
+  margin-left: 0;
+  margin-right: 10%;
+  flex: 1;
+  overflow: hidden;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.buttons-0 {
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  list-style-type: none;
+  margin-top: 5%;
+  margin-left: 0%;
+  margin-right: 10%;
+  width: 100%;
+  flex-wrap: wrap;
+}
+
+.buttons-0 > li > button {
+  border: 1px solid #8993a4;
+  padding: 10px;
+  background: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+}
+
+.css0 {
+  margin-left: 10%;
+}
+
+.html0 {
+  margin-left: 20%;
+}
+
+.bootstrap0 {
+  margin-left: 30%;
+}
+
+.ruby0 {
+  margin-left: 70%;
+}
+
+.see-project {
+  display: flex;
+  flex: 1;
+  width: 130px;
+  height: 48px;
+  position: relative;
+  margin-top: 20px;
+  background-color: #ff6b00;
+  margin-left: 0;
+  align-items: center;
+}
+
+.see-project > a {
+  margin-left: 20%;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  letter-spacing: 0.03em;
+  color: #fff;
+  align-self: center;
+}
+
+.see-project > a :hover {
+  border: 2px solid blue;
+  text-decoration: underline;
+  text-decoration-color: blue;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.btn:hover {
+  border: 2px solid blue;
+  text-decoration: underline;
+  text-decoration-color: blue;
+}
+
+/* profesional-art */
+
+.profesional-art-1 {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  justify-content: center;
+  position: relative;
+  width: 90vw;
+  height: 50vh;
+  background-image: url(image/background.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.H3 {
+  position: relative;
+  margin-top: 30%;
+  width: 90%;
+  height: 81px;
+  left: 10px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 28px;
+  line-height: 44px;
+  color: #fff;
+  overflow: hidden;
+}
+
+.p2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  margin-top: 5%;
+  margin-left: 10px;
+}
+
+.buttons-1 {
+  display: flex;
+  flex-direction: row;
+  flex: 1;
+  list-style-type: none;
+  margin-top: 5%;
+  margin-left: 0;
+  margin-right: 10%;
+  width: 100%;
+  flex-wrap: wrap;
+}
```

Comment:
[OPTIONAL]
- [ ] Let's change the color of the buttons to match that of the Figma design
![image](https://user-images.githubusercontent.com/29071984/170777405-b14baad6-5163-4494-9a2e-aa7629c665e4.png)


 [OPTIONAL]
- [ ] Also I see you have separate CSS for the buttons in the project sections. It'd be great to give them one class and style them as one. This will help keep you from repeating yourself.